Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM alpine:3.10
+FROM python:latest
```

Comment:
I think it would be better to pin to a specific version of Python. Could you change it to 3.9 for now?

Reply:
okey, not problem